x-pack/test/load/runner.ts

```diff
@@ -52,6 +53,15 @@ export async function GatlingTestRunner({ getService }: FtrProviderContext) {
   const log = getService('log');
 
   await withProcRunner(log, async (procs) => {
+    await procs.run('node build/index.js', {
+      cmd: 'node',
+      args: ['build/index.js'],
+      cwd: puppeteerProjectRootPath,
+      env: {
+        ...process.env,
+      },
+      wait: true,
```

Comment:
@dmlemeshko hey man, what does the `wait` mean here?

Reply:
@wayneseymour `wait:true` here is to make sure we do not proceed until `node build/index.js` command is finished. Quick important to control execution